msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md

```diff
@@ -152,7 +152,7 @@ When interacting in a channel, your bot should be smart about taking certain con
 
 The `channelData` object contains Teams-specific information and is the definitive source for team and channel IDs. You should cache and use these ids as keys for local storage.
 
-The `channelData` object is not included in messages in personal conversations since these take place outside of any channel.
+The `channelData` object include only `tenant.id` in messages for personal conversations since these take place outside of any channel.
```

Comment:
Looks like this change is not correct.
Please wait, we will confirm once and let you know.